bin/run_tests_on_host

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -e
+cd tests/check-legal-aid
+npm install
```

Comment:
Do we need to add local runs in this pull request? I'm a bit concerned about the level of duplication and overlap this is adding.

- The setup step is advertised as making the necessary steps to prepare running tests. However, `run` here has `npm install` which could fail with installation errors. I feel those would be misleading in a "running tests" step.
- I feel the test runner should be unaware of the context it's running in and should receive the URLs and browser as configuration.

Reply:
:thumbsup: